Makefile

```diff
@@ -47,6 +47,9 @@ KUBECTL?=kubectl
 KUBECONFIG?=./.kubeconfig
 HUB_KUBECONFIG?=./.hub-kubeconfig
 
+CLUSTER_MANAGER_INSTALL_MODE?=Default
```

Comment:
I added a new target, `deploy-spoke-detach`, which will have the least impact on the original code. Which one do you think is better?

Reply:
Good thinking. At last, the cluster-manager will also provide a makefile target like `deploy-cluster-manager-detached`. There will be one unified way to interact with users.

But at the moment, we suppose the "detached-cluster-manager" is not provided yet. This env var is for future use.

Reply:
`sed` is also used to replace the image. In 'apply-hub-cr' target. So I suppose it may not affect the 'deploy-xx' part.

Reply:
Remove this part, add to the next PR.